### PR TITLE
Update CHANGELOG entries across several sdks

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

**Comment by rlazo** [comment]
friendly ping

File: firebase-crashlytics-ndk/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # Unreleased
```
Comment (daymxn) [inline firebase-crashlytics-ndk/CHANGELOG.md:1]: What's this for?

File: firebase-ml-modeldownloader/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # Unreleased
 
+# 24.1.1
+* [fixed] Fixed an issue where `FirebaseModelDownloader.getModel` was throwing
+  `FirebaseMlException.PERMISSION_DENIED` when the model name was empty. It now
+  throws `FirebaseMlException.INVALID_ARGUMENT`
+  (#4157)
```
Comment (daymxn) [inline firebase-ml-modeldownloader/CHANGELOG.md:7]: Are these also supposed to be links btw?

**Review by daymxn — APPROVED** [review]
LGTM, although would you mind pinging me in gc with the tooling that interprets this?

**Pull request merged**